src/palindrome.rs

```diff
@@ -1,4 +1,5 @@
 // http://rosettacode.org/wiki/Palindrome_detection
+// http://rosettacode.org/wiki/Test_a_function
```

Comment:
I think he means that the `coverage` crate won't pick this comment up and mark the task as completed, which it won't. Feel free to create an issue and I can fix it when I get some time.